package.json

```
@@ -1,51 +1,21 @@
 {
 	"name": "directus-monorepo",
 	"private": true,
-	"workspaces": [
-		"./app",
-		"./api",
-		"./packages/*"
-	],
 	"scripts": {
+		"build": "pnpm --recursive --filter '!docs' run build",
 		"format": "prettier --write \"**/*.{md,y?(a)ml,json}\"",
 		"lint": "eslint .",
-		"test:blackbox": "jest tests -c tests-blackbox/jest.config.js",
-		"test:blackbox:watch": "jest tests -c tests-blackbox/jest.config.js --watch",
-		"posttest:blackbox:watch": "ts-node --project ./tests-blackbox/tsconfig.json --transpile-only ./tests-blackbox/setup/teardown.ts"
+		"test": "pnpm --recursive --filter '!tests-blackbox' test"
```

```suggestion
		"test": "pnpm --recursive --filter '!tests-blackbox' test",
		"blackbox": "pnpm --filter directus deploy --prod dist && pnpm --filter tests-blackbox test"
```

Can we add a top level blackbox command to simplify the running of blackbox testing?